src/status_im/multiaccounts/create/core.cljs

```diff
@@ -224,50 +222,65 @@
           :else {:db (assoc-in db [:intro-wizard :step]
                                (inc-step step))})))
 
+(defn get-selected-multiaccount [db]
+  (let [{:keys [selected-id multiaccounts]} (:intro-wizard db)]
+    (some #(when (= selected-id (:id %)) %) multiaccounts)))
+
+(defn prepare-default-account [{:keys [publicKey address]}]
+  {:public-key publicKey
+   :address    address
+   :color      colors/blue
+   :default?   true
```

Comment:
I guess we can go like this for now but I'd rather have  a default-account setting than this key

Reply:
yes we can, i decided to do this for now, to keep minimum changes in multiaccount object

Reply:
ok deal